docs/index.md

```diff
@@ -1,148 +1,128 @@
-# Integration Guide
+# Integration guide
 
-## API Key
+Pulse displays metrics that provide insights about the current and historic performance of your software delivery process. To calculate these metrics, Pulse must collect information from key events of your particular software development workflow.
 
-The API key identifies your organization and authorizes you to push events to Pulse.
+## Integrating with Pulse
 
-## Pushing Data
+Currently, the recommended way of integrating your workflow with Pulse is by using a CLI to send the necessary data to Pulse.
 
-Currently, we support two ways to push data: a CLI and an HTTP API.
+However, in some scenarios it may not be feasible to use the CLI to send data to Pulse, such as when reporting changes or incidents, or if you are sending data from providers that only support webhooks. For these situations, we provide an HTTP POST webhook on the following URL:
 
-### CLI
+`https://ingestion.acceleratedevops.net/v1/ingestion/<PROVIDER>?api_key=<API KEY>`
 
-#### Install
+There is no specified format for sending events using this webhook. However, make sure to include all the fields for the events documented below.
```

Comment:
We should clarify this a bit better (perhaps examples next to each even will help). How does Pulse infer the event from the webhook call? Is it from the fields that are sent in the POST body? :thinking:

Reply:
It doesn't. 😁 
We are assuming that we would have to manually look at the events received and then figure out how to parse them ourselves - for the time being.

Reply:
Ha, I see. I'll leave it like this for now, then.